docs/data/date-pickers/timezone/timezone.md

```diff
@@ -0,0 +1,218 @@
+| IANA standard zones | Example: `"Europe/Paris"`, `"America/New_York"`<br/>[List of all the IANA zones](https://timezonedb.com/time-zones)                                                                                                                                                                                                                                                                                            |
+|        Fixed offset | Example: `"UTC+7"`<br/>**Only available with Luxon**                                                                                                                                                                                                                                                                                                                                                           |
+
+{{"demo": "TimezonePlayground.js", "defaultCodeOpen": false}}
+
+## Usage with Day.js
+
+### Day.js and UTC
+
+To be able to use UTC, you have to enable the `utc` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+
+dayjs.extend(utc);
+
+function App() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DateTimePicker defaultValue={dayjs.utc('2022-04-17T15:30')} />
+    </LocalizationProvider>
+  );
+}
+```
+
+{{"demo": "DayjsUTC.js", "defaultCodeOpen": false}}
+
+### Day.js and timezones
+
+To be able to use timezones, you have to enable both the `utc` and the `timezone` plugin:
+
+```tsx
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.tz.setDefault('America/New_York');
+
+function App() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DateTimePicker defaultValue={dayjs.tz('2022-04-17T15:30')} />
+    </LocalizationProvider>
+  );
+}
+```
+
+{{"demo": "DayjsTimezone.js", "defaultCodeOpen": false}}
+
+:::info
+You can check out the documentation of the [dayjs timezone plugin](https://day.js.org/docs/en/timezone/timezone) for more details on how to manipulate the timezones.
+:::
+
+## Usage with Luxon
+
+### Luxon and UTC
+
```

Comment:
I'm not in favor of saying to people how to create a UTC date like this.
Because for every library, there is several ways of doing it depending on there use case.
Most people will probably never use `DateTime.utc` and instead use `DateTime.fromISO(dateStr, { zone: "UTC"})`

Reply:
That's a good point. To be clear on my intention, I think it's a good pattern to give some context for what we want to show on the code snippets, so I tried to describe the "interesting" parts of them.

> I'm not in favor of saying to people how to create a UTC date like this.

I believe the code snippets already say that, but without "written words". 
Perhaps we should be clearer when giving the context that we showcase one of several ways of doing it, but we really recommend checking out the librarys' docs for a broader understanding of their API. The info after the example can be easily missed, maybe it should be displayed before. Also, I think we have to repeat some information because people will likely scroll directly to their adapter section, where they'll need an independent contextualization.

Reply:
>  To be clear on my intention, I think it's a good pattern to give some context for what we want to show on the code snippets, so I tried to describe the "interesting" parts of them.

Fully agree with that

> Perhaps we should be clearer when giving the context that we showcase one of several ways of doing it, but we really recommend checking out the librarys' docs for a broader understanding of their API.

I think that's the information to give indeed "you must pass an UTC date, here is a way to do it, checkout the library doc to find out more".

>  Also, I think we have to repeat some information because people will likely scroll directly to their adapter section,

Fully agree with that as well, the info can be reproduced in each section.

Reply:
@joserodolfofreitas I updated the doc for Luxon
https://deploy-preview-8261--material-ui-x.netlify.app/x/react-date-pickers/timezone/#usage-with-luxon

I will apply the same approach on Moment and Day.js once we reach to something good for Luxon :+1: